docs/excel/excel-add-ins-ranges-set-get.md

```diff
@@ -47,6 +47,95 @@ Excel.run(function (context) {
+
+![A table with data from C5:F12 in Excel. The range D8:E9 is selected.](../images/excel-ranges-used-range-d8-e9.png)
+
+#### After selecting the cell at the edge of the used range
+
+The following screenshot shows the same table as the preceding screenshot, with data in the range **C5:F12**. Inside this table, the range **D5** is selected. This selection is *after* state, after running the `Range.getRangeEdge` method to select the cell at the edge of the used range in the up direction.
+
+![A table with data from C5:F12 in Excel. The range D5 is selected.](../images/excel-ranges-used-range-d5.png)
+
+### Select all cells from current range to furthest edge of used range
+
+The following code sample shows how use the `Range.getExtendedRange` method to to select all the cells from the currently selected range to the furthest edge of the used range, in the direction down. This action matches the result of using the Ctrl+Shift+Arrow key keyboard shortcut while a range is selected.
+
+```js
+Excel.run(function (context) {
+    // Get the selected range.
+    var range = context.workbook.getSelectedRange();
+
+    // Specify the direction with the `KeyboardDirection` enum.
+    var direction = Excel.KeyboardDirection.down;
+
+    // Get the active cell in the workbook.
+    var activeCell = context.workbook.getActiveCell();
+
+    // Get all the cells from the currently selected range to the bottom-most edge of the used range.
+    // This method acts like the Ctrl+Shift+Arrow key keyboard shortcut while a range is selected.
+    var extendedRange = range.getExtendedRange(
+      direction,
+      activeCell
+    );
+    extendedRange.select();
+
+    return context.sync();
+}).catch(errorHandlerFunction);
+```
+
+#### Before selecting all the cells from the current range to the edge of the used range
+
+The following screenshot shows a used range and a selected range within the used range. The used range is a table with data at **C5:F12**. Inside this table, the range **D8:E9** is selected. This selection is the *before* state, prior to running the `Range.getExtendedRange` method.
+
+![A table with data from C5:F12 in Excel. The range D8:E9 is selected.](../images/excel-ranges-used-range-d8-e9.png)
+
+#### After selecting all the cells from the current range to the edge of the used range
+
+The following screenshot shows the same table as the preceding screenshot, with data in the range **C5:F12**. Inside this table, the range **D8:E12** is selected. This selection is *after* state, after running the `Range.getExtendedRange` method to select all the cells from the current range to the edge of the used range in the down direction.
+
+![A table with data from C5:F12 in Excel. The range D8:E12 is selected.](../images/excel-ranges-used-range-d8-e12.png)
+
 ## See also
```

Comment:
Note: in this section is a link to the same article. In other words, the link "Set and get the selected range using the Excel JavaScript API" goes to this same article. Recommend removing.

Reply:
I don't see the issue you describe. The "See also" has a link to a similar article on line 54: "Set and get range values, text, or formulas using the Excel JavaScript API" which links to "excel-add-ins-ranges-set-get-values.md" 

What line is the issue you describe on?